### PR TITLE
Update unmaintained packages

### PR DESCRIPTION
**Pull request by matthiasbeyer**
**NOTE**: All these patches were fetched from monitor.nixos.org and were marked green, which means (AFAIK) that the patch builds.

I did _not_ build them locally, but I will now start building them on a VM I have in my universitys cloud and then report back.
###### Things done
- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [ ] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/tools/build-managers/gnumake/4.0/default.nix [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {stdenv, fetchurl}:
```
Comment (joachifm) [inline pkgs/development/tools/build-managers/gnumake/4.0/default.nix]: I don't think this makes any sense ...

File: pkgs/development/tools/misc/autoconf/2.64.nix [hunk 2]
```diff
@@ -1,11 +1,11 @@
 { stdenv, fetchurl, m4, perl }:
```
Comment (joachifm) [inline pkgs/development/tools/misc/autoconf/2.64.nix]: This doesn't make sense

File: pkgs/development/tools/build-managers/gnumake/4.0/default.nix (hunk 1 above)
Comment (matthiasbeyer) [inline pkgs/development/tools/build-managers/gnumake/4.0/default.nix]: What do you mean? I just updated the package, I did not alter the line you annotated!

File: pkgs/development/tools/misc/autoconf/2.64.nix (hunk 2 above)
Comment (matthiasbeyer) [inline pkgs/development/tools/misc/autoconf/2.64.nix]: Same here: What do you mean? I just updated the package, I did not alter the line you annotated!

File: pkgs/development/tools/build-managers/gnumake/4.0/default.nix (hunk 1 above)
Comment (joachifm) [inline pkgs/development/tools/build-managers/gnumake/4.0/default.nix]: You're updating gnumake40 to 4.1. Clearly, gnumake40 is supposed to be pinned to 4.0.x

File: pkgs/development/tools/misc/autoconf/2.64.nix (hunk 2 above)
Comment (joachifm) [inline pkgs/development/tools/misc/autoconf/2.64.nix]: You're updating a pinned package to a version beyond what it's supposed to be pinned to.

Comment (FRidh) [inline pkgs/development/tools/misc/autoconf/2.64.nix]: The file you updated is meant to correspond to exactly version 2.64; it should not be updated.

File: pkgs/development/tools/build-managers/gnumake/4.0/default.nix (hunk 1 above)
Comment (matthiasbeyer) [inline pkgs/development/tools/build-managers/gnumake/4.0/default.nix]: Whoops. Yes,... that happens when you just pull patches from monitor. I'll remove this patch.

File: pkgs/development/tools/misc/autoconf/2.64.nix (hunk 2 above)
Comment (matthiasbeyer) [inline pkgs/development/tools/misc/autoconf/2.64.nix]: Same here I assume?

File: pkgs/os-specific/linux/bluez/bluez5_28.nix [hunk 3]
```diff
@@ -5,11 +5,11 @@
 assert stdenv.isLinux;
 
 stdenv.mkDerivation rec {
-  name = "bluez-5.28";
+  name = "bluez-5.39";
```
Comment (FRidh) [inline pkgs/os-specific/linux/bluez/bluez5_28.nix]: same here

Comment (matthiasbeyer) [inline pkgs/os-specific/linux/bluez/bluez5_28.nix]: Will remove the patch.

**Comment by matthiasbeyer** [comment]
Removed the patches which updated pinned packages.

**Comment by matthiasbeyer** [comment]
So far, `libva-full` and `rabbitmq_server` failed on my build machine. The other packages are not yet build (I also build the patches from the branch before removing the pinned package update patches).

I will remove the patches which did not build on my build machine as soon as the build is finished.

File: pkgs/development/libraries/isl/0.15.0.nix [hunk 4]
```diff
@@ -1,11 +1,11 @@
 { stdenv, fetchurl, gmp }:
 
 stdenv.mkDerivation rec {
-  name = "isl-0.15";
```
Comment (dezgeg) [inline pkgs/development/libraries/isl/0.15.0.nix]: This one seems also pinned.

Comment (matthiasbeyer) [inline pkgs/development/libraries/isl/0.15.0.nix]: Will remove as well.

**Comment by matthiasbeyer** [comment]
Failed on my build machine:

* libva-full
* rabbitmq_server
* vlc
* impressive
* geis
* gemrb
* graphicsmagick137
* graphicsmagick_q16
* belle-sip
* bluez
* bluez5_28

Will remove these commits from the branch now.

**Comment by joachifm** [comment]
Would you mind moving the dbus update to a separate PR? Then I think we can merge.

**Comment by matthiasbeyer** [comment]
Removed dbus update patch

**Pull request merged**